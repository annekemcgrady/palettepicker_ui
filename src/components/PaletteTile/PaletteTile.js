import React from 'react';
import './PaletteTile.scss'

const PaletteTile = ({ isLocked, hexCode }) => {
    return (
        <article className='palette-tile'>
            <p>{hexCode}</p>


        </article>
    )
}

export default PaletteTile;