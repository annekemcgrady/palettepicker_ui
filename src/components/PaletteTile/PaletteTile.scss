.palette-tile {
  background-color: magenta;
  height: 300px;
  margin: 10px;
  width: 200px;
}

