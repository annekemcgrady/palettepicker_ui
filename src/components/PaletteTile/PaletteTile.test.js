import React from 'react';
import PaletteTile from './PaletteTile';
import { shallow } from 'enzyme';


describe('PaletteTile', () => {
  let wrapper;
  let instance;
  let props;




})