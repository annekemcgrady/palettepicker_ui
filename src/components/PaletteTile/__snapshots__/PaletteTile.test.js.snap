// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaletteTile should match the snapshot 1`] = `ShallowWrapper {}`;
